app/src/main/java/com/mercadolibre/android/mlbusinesscomponentsapp/touchpoint/TouchpointSamples.java

```diff
@@ -10,6 +10,7 @@
     FULL_GRID(R.raw.touchpoint_grid_full_content),
     PARTIAL_GRID(R.raw.touchpoint_grid_partial_content),
     CAROUSEL(R.raw.touchpoint_carousel),
+    CAROUSEL_TOUCHPOINT_V2(R.raw.touchpoint_carousel_touchpoint_v2),
```

Comment:
Una consulta porque no conozco, para qué es esto?

Reply:
Ah, yo pensé que esto venía de BE 😵

Reply:
Claro solo para la Testapp, te asusté 🤣

Reply:
Aaaaaaaaaah, ahí va jajaja, no me había dado cuenta de eso!